### PR TITLE
[Refactor] Culling test with Corrade::TestSuite

### PR DESCRIPTION
**Pull request by bigbike**
## Motivation and Context
Corrade::TestSuite provides a convenient way to do benchmark. I may need this functionality later.

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have completed my CLA (see **CONTRIBUTING**)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/tests/CullingTest.cpp [hunk 1]
```diff
@@ -46,7 +63,7 @@ TEST(CullingTest, computeAbsoluteAABB) {
       esp::assets::AssetInfo::fromPath(sceneFile);
   bool loadSuccess =
       resourceManager.loadScene(info, &sceneRootNode, &drawables);
-  EXPECT_EQ(loadSuccess, true);
+  CORRADE_COMPARE(loadSuccess, true);
```
Comment (mosra) [inline src/tests/CullingTest.cpp]:
```suggestion
  CORRADE_VERIFY(loadSuccess);
```

File: src/tests/CullingTest.cpp [hunk 2]
```diff
@@ -88,17 +105,17 @@ TEST(CullingTest, computeAbsoluteAABB) {
       {{3.0f, -1.0f, 3.0f}, {5.0f, 1.0f, 5.0f}}};
 
   // compare against the ground truth
-  EXPECT_EQ(aabbs.size(), aabbsGroundTruth.size());
-  const float epsilon = 1e-6;
+  CORRADE_COMPARE(aabbs.size(), aabbsGroundTruth.size());
+  const float eps = 1e-6;
   for (unsigned int iBox = 0; iBox < aabbsGroundTruth.size(); ++iBox) {
```
Comment (mosra) [inline src/tests/CullingTest.cpp]: You may want to use [`CORRADE_ITERATION()`](https://doc.magnum.graphics/corrade/classCorrade_1_1TestSuite_1_1Tester.html#TestSuite-Tester-iteration-annotations) here for better failure diagnostic:

```cpp
  for (unsigned int iBox = 0; iBox < aabbsGroundTruth.size(); ++iBox) {
    CORRADE_ITERATION(iBox);
    …
```

File: src/tests/CullingTest.cpp [hunk 3]
```diff
@@ -204,7 +221,7 @@ TEST(CullingTest, frustumCulling) {
     q.end();
     target->renderExit();
 
-    EXPECT_EQ(q.result<bool>(), false);
+    CORRADE_COMPARE(q.result<bool>(), false);
```
Comment (mosra) [inline src/tests/CullingTest.cpp]:
```suggestion
    CORRADE_VERIFY(!q.result<bool>());
```

File: src/tests/CullingTest.cpp [hunk 4]
```diff
@@ -119,7 +136,7 @@ TEST(CullingTest, frustumCulling) {
       esp::assets::AssetInfo::fromPath(sceneFile);
   bool loadSuccess =
       resourceManager.loadScene(info, &sceneRootNode, &drawables);
-  EXPECT_EQ(loadSuccess, true);
+  CORRADE_COMPARE(loadSuccess, true);
```
Comment (mosra) [inline src/tests/CullingTest.cpp]:
```suggestion
  CORRADE_VERIFY(resourceManager.loadScene(info, &sceneRootNode, &drawables));
```

(oh, cool, multi-line suggestions finally (at least partially) work? :O)

File: src/tests/CullingTest.cpp [hunk 5]
```diff
@@ -227,7 +244,7 @@ TEST(CullingTest, frustumCulling) {
         target->renderExit();
 
         // check if it a genuine visible drawable
-        EXPECT_EQ(q.result<bool>(), true);
+        CORRADE_COMPARE(q.result<bool>(), true);
```
Comment (mosra) [inline src/tests/CullingTest.cpp]:
```suggestion
        CORRADE_VERIFY(q.result<bool>());
```

File: src/tests/CullingTest.cpp [hunk 6]
```diff
@@ -26,7 +28,22 @@ namespace Mn = Magnum;
 using esp::assets::ResourceManager;
 using esp::scene::SceneManager;
 
-TEST(CullingTest, computeAbsoluteAABB) {
+namespace Test {
```
Comment (mosra) [inline src/tests/CullingTest.cpp]:
```suggestion
namespace Test { namespace {
```

Not strictly needed, but on GCC and Clang this causes useful warnings to be printed when you have accidentally unused variables or functions in the test, which I think is very useful.

**Comment by bigbike** [comment]
Great. just updated.
@mosra : Thank you for the feedback.

**Pull request merged**